src/lib/header-bidding/prebid/bid-config.ts

```diff
@@ -266,22 +267,27 @@ const ozoneClientSideBidder: (pageTargeting: PageTargeting) => PrebidBidder = (
 	bidParams: (
 		_slotId: string,
 		sizes: HeaderBiddingSize[],
-	): PrebidOzoneParams => ({
-		publisherId: 'OZONEGMG0001',
-		siteId: '4204204209',
-		placementId: getOzonePlacementId(sizes),
-		customData: [
-			{
-				settings: {},
-				targeting: {
-					// Assigns a random integer between 0 and 99
-					testgroup: String(Math.floor(100 * Math.random())),
-					...buildAppNexusTargetingObject(pageTargeting),
+	): PrebidOzoneParams => {
+		const advert = dfpEnv.adverts.find((ad) => ad.id === _slotId);
```

Comment:
@chrislomaxjones depending on who merges first I think this will affect/be affected by #1080

Reply:
Who should merge first? @arelra @chrislomaxjones

Reply:
I think lets get this one in first @deedeeh as it is more urgent. Thanks!